src/views/portals/outings-stats/OutingsStatsPart.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div class="columns result-section">
+      <div class="column">
+        <h2 class="title is-4" v-translate>By year</h2>
+        <div ref="year_repartition"></div>
+      </div>
+      <div class="column">
+        <h2 class="title is-4" v-translate>By month</h2>
+        <div ref="month_repartition"></div>
+      </div>
+    </div>
+
+    <div class="columns result-section">
+      <div class="column">
+        <h2 class="title is-4" v-translate>height_diff_up</h2>
+        <div ref="height_diff_up"></div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { Histogram } from './outings-stats';
+
+const getOutingYear = function (outing) {
+  return parseInt(outing.date_start.substring(0, 4), 10);
+};
+const getOutingMonth = function (outing) {
+  return parseInt(outing.date_start.substring(5, 7), 10) - 1;
+};
+
+const formatLengthInMeter = function (length) {
+  if (length < 9999) {
+    return `${length}m`;
```

Comment:
Thought the new standard was to always keep an unbreakable space before units?

Reply:
Totally right